### PR TITLE
Issue180

### PR DESCRIPTION
**Pull request by gamaanderson**
change display to remove colormap and manually move axes inside figure.

### CODE REVIEW

File: artview/components/plot_radar.py [hunk 1]
```diff
@@ -849,6 +856,65 @@ def _update_fig_ax(self):
         self.ax.set_position([0.2, 0.55-0.5*yheight, xwidth, yheight])
         self.cax.set_position([0.2, 0.10, xwidth, 0.02])
         self._update_axes()
+    '''
+        options_type = [
+            ("Plot     top    left",  (float, float)),
+            ("Plot     button right", (float, float)),
+            ("Colormap top    left",  (float, float)),
```
Comment (nguy) [inline artview/components/plot_radar.py]: 'button' should be 'bottom'

File: artview/components/plot_radar.py [hunk 2]
```diff
@@ -849,6 +856,65 @@ def _update_fig_ax(self):
         self.ax.set_position([0.2, 0.55-0.5*yheight, xwidth, yheight])
         self.cax.set_position([0.2, 0.10, xwidth, 0.02])
         self._update_axes()
+    '''
+        options_type = [
+            ("Plot     top    left",  (float, float)),
+            ("Plot     button right", (float, float)),
```
Comment (nguy) [inline artview/components/plot_radar.py]: Could we change all of the "Plot" to "Plot area" ? This will also help with the spacing.

**Comment by nguy** [comment]
This is super fun! Couple minor changes and we can merge.

**Comment by gamaanderson** [comment]
done

**Pull request merged**